src/components/Icon/Icon.tsx

```diff
@@ -51,22 +52,40 @@ class Icon extends UIComponent<any, any> {
     'circular',
     'className',
     'disabled',
-    'kind',
+    'font',
     'name',
     'size',
     'xSpacing',
   ]
 
   static defaultProps = {
-    as: 'i',
-    kind: 'FontAwesome',
+    as: 'span',
```

Comment:
Is there a reason you're changing this from the I tag? I is (or was) standard, for the most part.

Reply:
from MDN

> The HTML <i> element represents a range of text that is set off from the normal text for some reason. Some examples include technical terms, foreign language phrases, or fictional character thoughts. It is typically displayed in italic type.

`<span>` seems to be a more appropriate and neutral element, especially given that we have SVG-based icon as a default option.